### ISSUE TITLE
How to perform two WebHook actions at the same time?

### ISSUE THREAD

**Issue by marcovega** [def-studio/telegraph#57]
Hello! I've been using your package very heavily recently. Great job!

I have a question: I have a method called `setRole` that's hooked to a Telegram action. This is an example of the contents:

```
$this->chat->message("You selected $role_id.")->send();
$this->chat->deleteMessage($the_action_message_id);
```

Only the first command gets executed, if I swap send message for delete message, then the message gets deleted. Is there a way to make them both work?

Thanks!

**Comment by marcovega** [comment]
Code for context

```php
<?php

namespace App\Http\Webhooks;
use DefStudio\Telegraph\Handlers\WebhookHandler;

class CustomWebhookHandler extends WebhookHandler
{

    /* ... more code ... */

    public function setRole(): void
    {
        $this->auth(); // Custom method that works
        $role_selection_message = UserMeta::where('user_id', Auth::user()->id)->where('meta_key', 'role_selection_message_id')->first();

        $role_id = $this->data->get('id');

        // Only the first one executes, delete gets ignored. If delete goes first, then the message gets ignored.
        $this->chat->message("You selected $role_id")->send();
        $this->chat->deleteMessage($role_selection_message->meta_value);
    }
}

**Comment by marcovega** [comment]
After experimenting with the Keyboard fix: https://github.com/def-studio/telegraph/pull/59, now this structure works for: replying, sending a message, and removing the keyboard.

```php
public function setRole(): void
{    
    $this->chat->message('Some text message')->send();
    $this->deleteKeyboard();
    $this->reply('Some reply');
}
```

**Comment by fabio-ivona** [comment]
Hi @marcovega! Thanks for your kind words, I'm glad you found my package useful

I will check soon your commit and try to track down the issue

Could you paste here the result returned from the two calls that originated the failure?

**Comment by fabio-ivona** [comment]
seems fixed  by #59  

if any issue still comes up, pleas reopen this issue :wink:
